app/models/manager_refresh/inventory_collection/builder/infra_manager.rb

```diff
@@ -4,7 +4,8 @@ class Builder
       class InfraManager < ::ManagerRefresh::InventoryCollection::Builder
         def networks
           add_properties(
-            :manager_ref => %i(hardware ipaddress ipv6address)
+            :manager_ref                  => %i(hardware ipaddress ipv6address),
+            :parent_inventory_collections => %i(vms),
```

Comment:
Shouldn't most of these be `%i(vms miq_templates)` ?

Reply:
I was checking that ovirt sets those only for Vms. If VMware uses also miq_templates we should do %i(vms miq_templates)

Reply:
No VMware uses a different parent collection (vms_and_templates) but ovirt looks like it does the same parsing for both vms and templates, https://github.com/ManageIQ/manageiq-providers-ovirt/blob/master/app/models/manageiq/providers/redhat/inventory/parser/infra_manager.rb#L326
@borod108 ?

Reply:
@Ladas what's the worst that will happen if we include miq_templates and ovirt doesn't happen to use it?  I'm leaning towards just doing %w(vms miq_templates) so we can get this in

Reply:
yes, let me change this